keel-sql/src/test/resources/testcontainers.properties

```diff
@@ -1 +1 @@
-ryuk.container.image = public.ecr.aws/s4w6t4b6/testcontainers/ryuk:0.3.4
+ryuk.container.image = testcontainers/ryuk:0.11.0
```

Comment:
FYI the ECR image wasn't working in local testing - thus the need to switch.  PROBABLY should use docker.io/testcontainers/ryuk:0.11.0 but this works